src/arc-validate/APIs/ValidateAPI.fs

```diff
@@ -50,6 +50,36 @@ module ValidateAPI =
                 "   - run [green]arc-validate validate -p <your-desired-package-name>[/] to validate agaionst a validation package."
             ]
             |> List.iter AnsiConsole.MarkupLine
+            
+            let status = AnsiConsole.Status()
+            let mutable exitCode = ExitCode.Success
+
+            status.Start($"Performing validation against the ARC specification V2_Draft", fun ctx ->
+
+                if verbose then
+
+                    AnsiConsole.MarkupLine("LOG: Running in:")
+                    AnsiConsole.Write(TextPath(Path.GetFullPath(root)))
+                    AnsiConsole.MarkupLine("")
+                    AnsiConsole.MarkupLine($"LOG: running validation against [bold underline green]ARC specification V2_Draft[/].")
+                    AnsiConsole.MarkupLine($"LOG: Output path is:")
+                    AnsiConsole.Write(TextPath(Path.GetFullPath(outPath)))
+                    AnsiConsole.MarkupLine("")
+
+                let cases = ARCSpecification.V2_Draft.validationCases root 
+                
+                let outDirBadge = System.IO.Path.Combine(root, "ARC_specification_V2_Draft.svg")
+                let outDirResXml =System.IO.Path.Combine(root, "ARC_specification_V2_Draft.xml")
+
+                ARCExpect.Execute.ValidationPipeline(jUnitPath=outDirResXml, badgePath=outDirBadge,labelText="ARC specification V2_Draft") cases
```

Comment:
`Execute.ValidationPipeline` is a composite method that combines `Execute.Validation`, `Execute.JUnitSummaryCreation`, and `Execute.BadgeCreation`. Therefore you are validating against `cases` two times here. Since you need the runSumm**a**ry below to determine the exit code, please use the three separate functions i mentioned above.